boot/zephyr/prj.conf

```diff
@@ -11,4 +11,3 @@ CONFIG_MBEDTLS_CFG_FILE="config-boot.h"
 CONFIG_HEAP_MEM_POOL_SIZE=16384
 
 CONFIG_FLASH=y
-CONFIG_SOC_FLASH_STM32F4=y
```

Comment:
Did you do something to Zephyr to make this work? When I did it, there were two flash devices defined for the target so Kconfig picked neither.